spec/gibbon/gibbon_spec.rb

```diff
@@ -1,5 +1,7 @@
 require 'spec_helper'
 require 'cgi'
+require 'webmock'
+require 'webmock/rspec'
```

Comment:
Using rspec to stub net http response to return fake chunks of json did not work (`response.read_body do |chunk|`, export.rb line 33). One can, of course, use rspec to stub the response method to yield a different, fake, HTTPResponse at `http.request req do |response|` (export.rb, line 33). But, to prevent from having to clutter the test file with creating fake HTTPResponse objects, webmock gives an excellent tool that does this for you.

So, strictly speaking, it is not necessary. But it is preferred!